drivers/video/video.c

```diff
@@ -1180,9 +1202,51 @@ static int video_reqbufs(FAR struct video_mng_s         *vmng,
 
   leave_critical_section(flags);
 
+  if ((ret == OK) && (reqbufs->memory == V4L2_MEMORY_MMAP))
+    {
+      vmng->buf_count = reqbufs->count;
+      if (vmng->vmem_heap != NULL)
+        {
+          kmm_free(vmng->vmem_heap);
```

Comment:
let's use kmm_realloc instead kmm_free/kmm_malloc

Reply:
Can realloc ensure alignment of address similarly to memalign?